sql/expression/function/greatest_least_test.go

```diff
@@ -81,6 +81,22 @@ func TestGreatest(t *testing.T) {
 			},
 			"bbb",
 		},
+		{
+			"nulls of a non-null type, char",
+			[]sql.Expression{
+				expression.NewConvert(expression.NewLiteral("aaa", sql.LongText), expression.ConvertToChar),
+				expression.NewConvert(expression.NewLiteral(nil, sql.Null), expression.ConvertToChar),
```

Comment:
Can you just use nil instead of a literal for the nil? Does it change the behavior?